src/GameShelf.js

```diff
@@ -54,30 +54,23 @@ export default class GameShelf extends Component {
     })
   } 
   filterByMulti = (games) => {
-    return games.filter(game => game.multiplayer.toString().includes(this.state.isMulti))
+    return games.filter(game => game.multiplayer === this.state.isMulti)
   }
   shuffle = (games) => {
     let randoArr = games.sort(() => 0.5 - Math.random());
-    return [randoArr[0]] || [];
+    return [randoArr[0]];
   }
 
   render () {
     let games = this.props.games;
-    if (this.state.titleFilter){
-      games = this.filterGamesTitle(games)
-    }
-    if (this.state.genres.length){
-      games = this.filterByKey(games, 'genres')
-    }
-    if (this.state.platforms.length){
-      games = this.filterByKey(games, 'platforms')
-    }
-    if (this.state.isLucky && games.length) {
-      games = this.shuffle(games)
-    }
-    if (this.state.isMulti !== null){
-      games = this.filterByMulti(games)
-    }
+
+    games = this.state.titleFilter ? this.filterGamesTitle(games) : games;
```

Comment:
lines 67-72 look awesome. So much cleaner to have ternaries rather than if statements